streamer/src/quic.rs

```diff
@@ -300,7 +439,7 @@ mod test {
         let (sender, _receiver) = unbounded();
         let keypair = Keypair::new();
         let ip = "127.0.0.1".parse().unwrap();
-        let t = spawn_server(s, &keypair, ip, sender, exit.clone()).unwrap();
+        let t = spawn_server(s, &keypair, ip, sender, exit.clone(), 1).unwrap();
```

Comment:
Perhaps we might want to have this be a small constant over 1 (for example 8 or 16) to e.g. accommodate clients behind NATs. As well, while I don't think there are any places right now where this happens, in certain cases, it may not be possible to guarantee in the quic-client that there is only at most 1 connection - depending on how the quic-client is used, it may be possible for it to have a small number of multiple concurrent connections for a short time.

Reply:
Yea, nat is fair.